components/team/screens/Team.tsx

```diff
@@ -57,9 +57,9 @@ const Team = ({ team }) => {
     }
 
     return (
-      <section className="mx-auto min-w-full lg:min-w-lg max-w-5xl grid grid-cols-1 sm:grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-x-12 gap-y-6">
+      <section className="mx-auto min-w-full lg:min-w-lg max-w-5xl flex flex-wrap gap-x-12 gap-y-6 justify-center">
         {members.map((member) => {
-          return <Member key={member.id} member={member} />;
+          return member.user.username !== null && <Member key={member.id} member={member} />;
```

Comment:
Should this be possible? Wondering if the `username` should be nullable - I think it should be required.